os/include/net/lwip/sockets.h

```diff
@@ -412,9 +412,13 @@ typedef struct ip_mreq {
 #define _IOW(x, y, t)     (IOC_IN | (((long)sizeof(t) & IOCPARM_MASK) << 16) | ((x) << 8) | (y))
 #endif							/* !defined(FIONREAD) || !defined(FIONBIO) */
 
+/* FIONREAD is already defined in tinyara/fs/ioctl.h.
+   To avoid conflict, we disable this command. */
+#if 0
```

Comment:
Can't we just remove the lines instead of adding #if 0 ?

Reply:
I'll remove the lines because it's not used.